detection_rules/integrations.py

```diff
@@ -213,7 +213,8 @@ def find_latest_compatible_version(package: str, integration: str,
     raise ValueError(f"no compatible version for integration {package}:{integration}")
 
 
-def get_integration_manifests(integration: str, prerelease: str, kibana_version: str) -> list:
+def get_integration_manifests(integration: str, prerelease: Optional[str] = "false",
+                              kibana_version: Optional[str] = "") -> list:
```

Comment:
This was necessary so that prerelease and kibana_version are optional and do not return errors from commands such as `python -m detection_rules dev integrations build-schemas -o`